events/triggers/debug

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Function to upload the output to hastebin
+# Source: https://github.com/seejohnrun/haste-client#lightweight-alternative
+haste() { a=$(cat); curl -X POST -s -d "$a" https://hastebin.com/documents | awk -F '"' '{print "https://hastebin.com/"$4}'; }
+
+UMBREL_ROOT="$(readlink -f $(dirname "${BASH_SOURCE[0]}")/../..)"
+DEBUG_FILE="${UMBREL_ROOT}/scripts/debug"
```

Comment:
I think DEBUG_SCRIPT would be a better name, DEBUG_FILE sounds like a file the output of the script is stored in.

Reply:
Yeah you're right, gonna change this

Reply:
Resolved in 589207a.